tox.ini

```diff
@@ -1,7 +1,6 @@
 [tox]
 envlist =
-    py26,
-    py27,
+    py37,
     flake8
```

Comment:
Something we've been doing now is changing our `flake8` env to `py37-flake8` just to make sure we're using flake8 with the right version of Python. Otherwise, it's possible that locally you might run the flake8 tests from Python 2.7.

Reply:
Would also need to change that name everywhere else, like further below and in the Travis config file.

Reply:
I think I changed things where they are needed.